deploy/csi-config-map-sample.yaml

```diff
@@ -79,6 +84,7 @@ data:
           "netNamespaceFilePath": "<kubeletRootPath>/plugins/cephfs.csi.ceph.com/net",
           "kernelMountOptions": "<kernelMountOptions for cephFS volumes>",
           "fuseMountOptions": "<fuseMountOptions for cephFS volumes>"
```

Comment:
add a comma to above fuse mountOptions at the end or else json parsing will have problem